shogi/CSA.py

```diff
@@ -63,8 +64,13 @@
 class Parser:
     @staticmethod
     def parse_file(path):
-        with open(path) as f:
-            return Parser.parse_str(f.read())
+        for enc in ['cp932', 'utf-8-sig']:
+            try:
+                with codecs.open(path, 'r', enc) as f:
+                    return Parser.parse_str(f.read())
+            except:
+                pass
+        return None
```

Comment:
It would be great if the function raises an exception when we cannot decode as `cp932` and `utf-8-sig` instead of returning `None`.